src/Arcus.WebApi.Tests.Unit/Hosting/Formatting/AzureFunctionsJsonFormattingMiddlewareTests.cs

```diff
@@ -131,6 +131,42 @@ public async Task Request_WithAllAllowAndJsonContentTypeHeaders_ReturnsOk()
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
         }
 
+        [Fact]
+        public async Task Request_WithJsonAllowHeaderWithExtension_ReturnsOk()
+        {
+            // Arrange
+            var middleware = new AzureFunctionsJsonFormattingMiddleware();
+            var context = TestFunctionContext.Create(req =>
+            {
+                req.Headers.TryAddWithoutValidation("allow", "application/json, q=4");
```

Comment:
q value can only range between 0 or 1 :)  Ok, this is a test so it doesn't matter in fact.
See also here on how it is used:
https://restfulapi.net/q-parameter-in-http-accept-header/

Reply:
Woops, 😅

Reply:
I'll change that.

Reply:
Done in https://github.com/arcus-azure/arcus.webapi/pull/406/commits/8cd193eb9bf81703f1aa2f5a53cc436e8869ef94